pkg/row/copier.go

```diff
@@ -275,7 +277,13 @@ func (c *Copier) GetETA() string {
 	// c.getCopyStats() and rowsPerSecond change estimation method when the PK is auto-inc.
 	remainingRows := totalRows - copiedRows
 	remainingSeconds := math.Floor(float64(remainingRows) / float64(rowsPerSecond))
-	return time.Duration(remainingSeconds * float64(time.Second)).String()
+
+	estimate := time.Duration(remainingSeconds * float64(time.Second))
+	comparison := c.CopierEtaHistory.AddCurrentEstimateAndCompare(estimate)
```

Comment:
Same comment about making functions private.

I think this is thread safe because there is a mutex on `GetETA()`, but just curious if you've put any thought into it?

Reply:
I did not, but it's easy enough